src/js/components/tasks/cards/index.tsx

```diff
@@ -160,6 +165,18 @@ const OrgCards = ({
     }
   };
 
+  const handleRefresh = (type: OrgTypes) => {
+    // set refetching org to true
+    setIsRefreshingOrg({ ...isRefreshingOrg, [type]: true });
+    // dispatch refetchObject
+    dispatch(refreshOrg(orgs[type])).finally(() => {
+      /* istanbul ignore else */
+      if (isMounted.current) {
+        // finally, refetching org to false
+        setIsRefreshingOrg({ ...isRefreshingOrg, [type]: false });
+      }
+    });
+  };
```

Comment:
I think this could move up and be written basically identical to `checkForOrgChanges` -- just accepts an `org` and calls `refreshOrg` with it.

Creation and deletion are more complex, for some reasons I understand and some reasons I completely forget. 😄 But I think refreshing can be simplified, more similar to checking for unsaved changes.